### PR TITLE
Tweak: Update context menu design [ED-11294]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: assets/dev/scss/editor/_context-menu.scss [hunk 1]
```diff
@@ -1,28 +1,32 @@
 .elementor-context-menu {
 	position: absolute;
-	width: 200px;
+	width: 250px;
+	padding: 5px 0px;
 	background-color: var(--e-a-bg-default);
 	border: var(--e-a-border);
-	border-radius: 3px;
+	border-radius: var(--e-a-border-radius);
 	box-shadow: 0 2px 20px 0 rgba(0, 0, 0, 0.1);
```
Comment (arielk) [inline assets/dev/scss/editor/_context-menu.scss:8]: Use --e-a-popover-shadow

File: assets/dev/scss/editor/_context-menu.scss [hunk 2]
```diff
@@ -40,12 +44,17 @@
 			}
 
 			&__icon {
```
Comment (arielk) [inline assets/dev/scss/editor/_context-menu.scss:46]: Why you are now remove the code completely instead of to hide via CSS?

**Review by arielk — CHANGES_REQUESTED** [review]
I left a few comments

File: assets/dev/scss/editor/_context-menu.scss (hunk 1 above)
Comment (rami-elementor) [reply assets/dev/scss/editor/_context-menu.scss]: Done.

File: assets/dev/scss/editor/_context-menu.scss [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
 	position: absolute;
-	width: 200px;
+	width: 250px;
+	padding: 5px 0px;
 	background-color: var(--e-a-bg-default);
-	border: var(--e-a-border);
-	border-radius: 3px;
-	box-shadow: 0 2px 20px 0 rgba(0, 0, 0, 0.1);
+	border-radius: var(--e-a-border-radius);
+	box-shadow: 0 4px 24px 0 rgba(0, 0, 0, 0.1);
```
Comment (arielk) [inline assets/dev/scss/editor/_context-menu.scss:7]: remove line

**Review by arielk — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/_context-menu.scss (hunk 2 above)
Comment (rami-elementor) [reply assets/dev/scss/editor/_context-menu.scss:46]: This PR simply hides icons, it doesn't remove the functionality.

**Review by arielk — APPROVED** [review]

**Pull request merged**